src/game/server/client.cpp

```diff
@@ -1237,9 +1237,6 @@ void CC_God_f (void)
 	   if ( gpGlobals->deathmatch )
```

Comment:
Remove the check for TF2 too (I'm not sure why these checks exist there too... since `buddha` is allowed)